src/storage/database_sink.rs

```diff
@@ -154,6 +154,8 @@ impl<'s> DatabaseSink<'s> {
             ));
         }
 
+        fields.push("geom geometry(Point)".to_owned());
```

Comment:
In line 128 we initialize it with `vec![]`, you can just add the geom as first field there.

Reply:
dbeeec65b9484ae89be714f6a9106092ada0a7ec

Reply:
Nooo, you still need two fields :wink:.

```
    let mut fields = vec![
        format!(
            "{} int not null",
            self.database_settings.surrogate_key_column,
        ),
        "geom geometry(Point)".to_string(),
    ];
```

Reply:
you don't have to go all darth vader on me. 
1. it still worked
2. the error is because postgis extension is missing in ci

5cdfffc3145002e6eb923bc4f18b3431b0df765f